test/blackbox-tests/test-cases/ppx-cross-context-issue.t/run.t

```diff
@@ -7,6 +7,14 @@
   Error: Library "fooppx" in _build/cross-environment/ppx is hidden
   (unsatisfied 'enabled_if').
   -> required by lib/lib.pp.ml (context cross-environment)
+  -> required by lib/.foolib.objs/lib.pp.ml.d (context cross-environment)
+  -> required by lib/.foolib.objs/foolib__Lib.impl.all-deps (context
+     cross-environment)
+  -> required by lib/.foolib.objs/byte/foolib__Lib.cmi (context
+     cross-environment)
+  -> required by lib/.foolib.objs/native/foolib__Lib.cmx (context
+     cross-environment)
+  -> required by lib/foolib.a (context cross-environment)
```

Comment:
Not sure if the new error is an improvement (more precise) or regression (contains too much internal information).

Reply:
Looks like a regression. Though might be difficult to fix.

Reply:
It's now fixed with the changed we discussed